cirq-google/cirq_google/engine/engine_client_test.py

```diff
@@ -1229,3 +1270,71 @@ def test_list_time_slots(client_constructor):
 
     client = EngineClient()
     assert client.list_time_slots('proj', 'processor0') == results
+
+
+@mock.patch.object(quantum, 'QuantumEngineServiceAsyncClient', autospec=True)
+def test_run_job_over_stream(client_constructor):
```

Comment:
"Test names should summarize the behavior being tested and its expected outcome." go/unit-testing-practices#naming